src/main/java/frc/robot/commands/TeleopSwerve.java

```diff
@@ -38,16 +38,18 @@ public TeleopSwerve(Swerve s_Swerve, DoubleSupplier translationSup, DoubleSuppli
         this.strafeSup = strafeSup;
         this.rotationSup = rotationSup;
         this.robotCentricSup = robotCentricSup;
-        this.dampen = dampen;
-        this.speedDial = speedDial;
+        this.dampen = dampen.getAsBoolean();
+        this.speedDial = speedDial.getAsDouble();
+
+        
     }
 
     @Override
     public void execute() {
         /* Get Values, Deadband*/
```

Comment:
When I originally meant was changing the code to look like this:
(in variables)
private BooleanSupplier dampenSupl;
private DoubleSupplier speedDialSupl;

(in constructor)
dampenSupl = dampen;
speedDialSupl = speedDial;

(here in execute())
float dampenFactor = dampenSupl.getAsBoolean() ? 0.2 : 1;
float speedDialFactor = ((speedDial.getAsDouble() + 1) / 2);
float speedScale = dampenFactor * speedDialFactor;

double translationVal = MathUtil.applyDeadband(translationSup.getAsDouble(), Constants.stickDeadband) * speedScale;
        double strafeVal = MathUtil.applyDeadband(strafeSup.getAsDouble(), Constants.stickDeadband) * speedScale;
        double  rotationVal = MathUtil.applyDeadband(rotationSup.getAsDouble(), Constants.stickDeadband) * speedScale;
        
        
        This approach of putting your common factors and calculations into named variables, allows you to do the calculation once, and simplifies the reading of the code where it is applied.